kivy/tests/async_common.py

```diff
@@ -220,7 +220,7 @@ def get_widget_pos_pixel(self, widget, positions):
     async def do_touch_down_up(
             self, pos=None, widget=None, duration=.2, pos_jitter=None,
             widget_jitter=False, jitter_dt=1 / 15., end_on_pos=False):
-        x, y = pos or widget.center
+        x, y = pos or widget.to_window(*widget.to_local(*widget.center))
```

Comment:
In that case, this part should be `x, y = pos or widget.to_window(*widget.center)` ?